.lintstagedrc.json

```diff
@@ -1,4 +1,4 @@
 {
-    "*": "prettier --write",
+    "*": "prettier --write --ignore-unknown",
```

Comment:
Ignore Unknown to prevent prettier to fail when unknown file extension is detected